app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -27,6 +27,20 @@ const EMOJI_LIST_OFFSET_LEFT = 8;
 const EMOJI_LIST_OFFSET_TOP = 8;
 const QUERY_MIN_LENGTH = 1;
 
+// DO NOT USE COLON WITH LOWERCASE LETTERS IN THE KEYS, or you will prevent searching emojis.
+// For example, while :D should be replaced with unicode symbol, :d should allow searching for :dancer:
+const EMOJI_INLINE_REPLACEMENT = {
```

Comment:
Hey @maximbaz ! Thanks for the commitment, team loves the new emojis logic!
Small 5 cents: there is already a list of emoticons that iOS client is substituting in the text, it can be potentially used for web too: https://github.com/wireapp/wire-ios/blob/develop/Wire-iOS/Sources/emoticons.min.json